android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.mkloarkuglll.crypto_reminder">
 
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Something simillar will probably have to be added for the iOS..

Reply:
Yep, I totally forgot about the iOS part..

Reply:
I don't find any documentation about adding permission to the iOS part...

Reply:
No worries, we can handle that later.